_haikus/natural_cloud

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Ensure that your filename ends in `.md`, so for this you should update your file to be called `natural_cloud.md`